relay-wstring/src/lib.rs

```diff
@@ -396,10 +424,10 @@ fn is_leading_surrogate(code_unit: u16) -> bool {
 /// These surrogate code unites have the first 6 bits set to a fixed prefix identifying
 /// whether tye are the *leading* or *trailing* code unit of the surrogate pair.  Anf for
 /// the trailing surrogate this bit prefix is `110111`, thus all trailing surrogates have a
-/// code unit between 0xDC00-0xDFFF.
+/// code unit of `0xDCxx`.
```

Comment:
Same here, the original wording was correct and describes what the code does now.